spec/controllers/wizard/steps/additional_code_controller_spec.rb

```diff
@@ -0,0 +1,56 @@
+RSpec.describe Wizard::Steps::AdditionalCodesController do
+  before do
+    allow(UserSession).to receive(:new).and_return(session)
+  end
+
+  let(:session) { build(:user_session, :with_commodity_information) }
+
+  describe 'GET #show' do
+    subject(:response) { get :show, params: { measure_type_id: '105' } }
+
+    it 'assigns the correct step' do
+      response
+      expect(assigns[:step]).to be_a(Wizard::Steps::AdditionalCode)
```

Comment:
Does `assigns(:step)` not work here?

Reply:
Annoying not. I can't put the fetching the response in the before block either as it creates a side affect that breaks other specs

Reply:
I tried

```ruby
expect(response; assigns[:step])
```

but that is weird/horrible